docker-compose.yml

```diff
@@ -16,6 +16,9 @@ services:
       ES_JAVA_OPTS: "-Xmx256m -Xms256m"
     networks:
       - elk
+    mem_limit: 4000000000
```

Comment:
Note to self: undo this, let elastic eat other people's computers, just not mine.